power-supply/power_supply.cpp

```diff
@@ -89,24 +89,6 @@ PowerSupply::PowerSupply(const std::string& name, size_t inst,
     powerOnTimer(e, std::bind([this]() { this->powerOn = true; }))
 {
     using namespace sdbusplus::bus;
-    using namespace witherspoon::pmbus;
-    std::uint16_t statusWord = 0;
-    try
-    {
-        // Read the 2 byte STATUS_WORD value to check for faults.
-        statusWord = pmbusIntf.read(STATUS_WORD, Type::Debug);
-        if (!((statusWord & status_word::INPUT_FAULT_WARN) ||
-              (statusWord & status_word::VIN_UV_FAULT)))
-        {
-            resolveError(inventoryPath,
-                         std::string(PowerSupplyInputFault::errName));
-        }
-    }
-    catch (ReadFailure& e)
-    {
-        log<level::INFO>("Unable to read the 2 byte STATUS_WORD value to check "
-                         "for power-supply input faults.");
-    }
```

Comment:
Why do you remove these?

Reply:
@alvintpwang I found that these are not available in ibm-openbmc/master version, and this section already exists in analyze ().
So I want to remove this paragraph if the master version is not available.

Reply:
Hmm, I thought this fix went into 940.  I will follow up.

Reply:
OP940 branch has this and your change should based on latest OP940 branch and don't remote these.